src/common/transaction/rbtccoin.js

```diff
@@ -1,5 +1,75 @@
 import Rsk3 from '@rsksmart/rsk3';
 import _ from 'lodash';
+import BigNumber from 'bignumber.js';
+import { NETWORK, ASSETS_CONTRACT } from '../constants';
+import assetAbi from '../assetAbi.json';
+
+const { MAINNET, TESTNET } = NETWORK;
+
+export const getTransactionFees = async (type, address, toAddress, fee, memo = '') => {
+  const rskEndpoint = type === 'Mainnet' ? MAINNET.RSK_END_POINT : TESTNET.RSK_END_POINT;
+  const rsk3 = new Rsk3(rskEndpoint);
+  const from = Rsk3.utils.toChecksumAddress(address);
+  const to = Rsk3.utils.toChecksumAddress(toAddress);
+  const gas = await rsk3.estimateGas({
+    from, to, value: fee, data: memo,
+  });
+  const latestBlock = await rsk3.getBlock('latest');
+  const { minimumGasPrice } = latestBlock;
+  const miniGasPrice = new BigNumber(minimumGasPrice, 16);
+  return {
+    gas: 40000,
+    gasPrice: {
+      low: miniGasPrice.toString(),
+      medium: miniGasPrice.multipliedBy(2).toString(),
+      high: miniGasPrice.multipliedBy(4).toString(),
+    },
+  };
+};
+
+export const getContractAddress = async (symbol, type) => {
+  if (ASSETS_CONTRACT[symbol] && ASSETS_CONTRACT[symbol][type]) {
+    const contractAddress = ASSETS_CONTRACT[symbol][type];
+    return Rsk3.utils.toChecksumAddress(contractAddress);
+  }
+  return '';
+};
+
+export const encodeContractTransfer = async (contractAddress, type, from, to, value) => {
+  const rskEndpoint = type === 'Mainnet' ? MAINNET.RSK_END_POINT : TESTNET.RSK_END_POINT;
+  const rsk3 = new Rsk3(rskEndpoint);
+  const contract = rsk3.Contract(assetAbi, Rsk3.utils.toChecksumAddress(contractAddress));
+  const data = await contract.methods.transfer(to, value).encodeABI();
+  return data;
+};
+
+export const createRawTransaction = async ({
+  symbol, type, sender, receiver, value, memo, gasPrice, gas, contractAddress,
+}) => {
+  const rskEndpoint = type === 'Mainnet' ? MAINNET.RSK_END_POINT : TESTNET.RSK_END_POINT;
+  const rsk3 = new Rsk3(rskEndpoint);
+  const from = Rsk3.utils.toChecksumAddress(sender);
+  const to = Rsk3.utils.toChecksumAddress(receiver);
+  const nonce = await rsk3.getTransactionCount(from, 'pending');
+  const rawTransaction = {
+    from,
+    nonce,
+    chainId: type === 'Mainnet' ? MAINNET.NETWORK_VERSION : TESTNET.NETWORK_VERSION,
+    gas,
+    gasPrice,
+    value,
+  };
+  if (symbol === 'RBTC' || !contractAddress) {
```

Comment:
This if should only check `symbol === 'RBTC' `

if symbol !== 'RBTC' && !contractAddress, the app should throw an error.

"Contract address for {symbol} is undefined."